app/controllers/dashboard_controller.rb

```diff
@@ -62,7 +62,8 @@ def profile_view
     @user = User.find(profile_params[:taskee_id])
   end
 
-  def assign_task
+  def profile_view
```

Comment:
Method `DashboardController#profile_view` is defined at both /app/app/controllers/dashboard_controller.rb:61 and /app/app/controllers/dashboard_controller.rb:65.